pkgs/development/python-modules/crewai/default.nix

```diff
@@ -0,0 +1,381 @@
+    "test_before_kickoff_with_none_input"
+    "test_multiple_before_after_kickoff"
+    "test_after_kickoff_modification"
+    "test_internal_crew_with_mcp"
+
+    # Tests task
+    "test_output_json_hierarchical"
+    "test_output_pydantic_sequential"
+    "test_no_inject_date"
+    "test_increment_tool_errors"
+    "test_task_execution_times"
+    "test_custom_converter_cls"
+    "test_save_task_output"
+    "test_inject_date"
+    "test_output_pydantic_hierarchical"
+    "test_json_property_without_output_json"
+    "test_task_interpolation_with_hyphens"
+    "test_increment_delegations_for_hierarchical_process"
+    "test_save_task_json_output"
+    "test_output_pydantic_to_another_task"
+    "test_output_json_dict_sequential"
+    "test_inject_date_custom_format"
+    "test_output_json_sequential"
+    "test_output_json_to_another_task"
+    "test_increment_delegations_for_sequential_process"
+    "test_save_task_pydantic_output"
+    "test_output_json_dict_hierarchical"
+    "test_task_with_max_execution_time"
+    "test_task_with_max_execution_time_exceeded"
+
+    # Tests task guardrails
+    "test_task_guardrail_process_output"
+    "test_guardrail_when_an_error_occurs"
+    "test_guardrail_emits_events"
+
+    # Tests tools
+    "test_delegate_work"
+    "test_delegate_work_with_wrong_co_worker_variable"
+    "test_ask_question_with_wrong_co_worker_variable"
+    "test_ask_question"
+    "test_delegate_work_withwith_coworker_as_array"
+    "test_ask_question_with_coworker_as_array"
+    "test_max_usage_count_is_respected"
+    "test_async_tool_using_within_isolated_crew"
+    "test_async_tool_using_decorator_within_flow"
+    "test_async_tool_using_decorator_within_isolated_crew"
+    "test_async_tool_within_flow"
+    "test_ainvoke"
+
+    # Tests tracing
+    "test_trace_listener_disabled_when_env_false"
+    "test_trace_listener_with_authenticated_user"
+    "test_trace_listener_collects_crew_events"
+    "test_trace_listener_ephemeral_batch"
+    "test_events_collection_batch_manager"
+
+    # Tests utilities
+    "test_converter_with_llama3_2_model"
+    "test_converter_with_llama3_1_model"
+    "test_converter_with_nested_model"
+    "test_convert_with_instructions"
+    "test_crew_emits_start_kickoff_event"
+    "test_crew_emits_test_kickoff_type_event"
+    "test_crew_emits_start_task_event"
+    "test_llm_emits_call_started_event"
+    "test_register_handler_adds_new_handler"
+    "test_multiple_handlers_for_same_event"
+    "test_llm_emits_event_with_lite_agent"
+    "test_tools_emits_finished_events"
+    "test_agent_emits_execution_started_and_completed_events"
+    "test_crew_emits_end_kickoff_event"
+    "test_llm_emits_stream_chunk_events"
+    "test_tools_emits_error_events"
+    "test_crew_emits_end_task_event"
+    "test_stream_llm_emits_event_with_task_and_agent_info"
+    "test_llm_no_stream_chunks_when_streaming_disabled"
+    "test_llm_emits_event_with_task_and_agent_info"
+
+    # Tests qdrant
```

Comment:
Please, provide the actual reason why these tests are failing ("require internet", "require unpackaged foo", etc.)

Reply:
Hi,

It's planned

Best Regards